### ISSUE TITLE
Edit to Wiki: installation

### ISSUE THREAD

**Issue by MizouziE** [academico-sis/academico#293]
Hi, 

I was goin through the repo as I would like to contribute to the project and I noticed one small error in the installation guide:

![image](https://user-images.githubusercontent.com/90829439/159466907-726aa9d0-b620-4037-a177-33ff0691a2eb.png)

The username should only be `admin` rather than the full email address shown above.

Not a big deal, just may be a gotcha for less experienced devs wanting to contribute. (Yes, it stumped me for a minute longer than I'd like to admit 😅 )

**Comment by thdebay** [comment]
You are right, the seeder was updated and I forgot to change it in the wiki. I have now updated the page. Thanks for reporting this issue!

As you might have noticed, the default branch of this repo is `oss` because it uses the free and open-source version of Backpack for Laravel, the package this entire application is built upon. Previously, Backpack for Laravel was a paid package, but they have recently been offering a free and open-source package with limited features. So I disabled the paid features that were used by Academico (that's the difference between the `oss` and the `pro` branches) but I haven't had the chance to test it extensively. Also, the documentation might not reflect these recent changes. Your input is appreciated is you notice something that needs to be updated!
